api/src/pcapi/routes/institutional/playlist.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import orm as sqla_orm
+
+from pcapi.core.criteria import models as criteria_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import models as offers_models
+from pcapi.routes.apis import public_api
+from pcapi.serialization.decorator import spectree_serialize
+from pcapi.validation.routes.institutional import require_institutional_api_token
+
+from . import serializers
+
+
+@public_api.route("/institutional/playlist/<tag_name>", methods=["GET"])
+@require_institutional_api_token
+@spectree_serialize(response_model=serializers.OffersResponse)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Avec ça tu dois déjà avoir un spectree qui existe (on en parlait tout à l'heure)